src/components/CertificateDropZone/CertificateDropZoneContainer.js

```diff
@@ -82,14 +111,17 @@ const mapStateToProps = store => ({
 const mapDispatchToProps = dispatch => ({
   updateNetworkId: () => dispatch(updateNetworkId()),
   updateCertificate: payload => dispatch(updateCertificate(payload)),
-  resetData: () => dispatch(resetCertificateState())
+  resetData: () => dispatch(resetCertificateState()),
+  processQr: payload => dispatch(processQrCode(payload))
 });
 
-export default connect(
+const ConnectedCertificateDropZoneContainer = connect(
```

Comment:
why this change ? its the same just a line extra.

Reply:
Same reason as above.
- Default export the connected version
- Named export for the non-connected version for unit test (we are not testing the connect feature and it messes around with the test)